default_down_quaternion: {
  roll: rad(0.0),
  pitch: rad(pi),
  yaw: rad(pi/4 + pi/2)
}

stow_joint_values:
  - 0.0
  - -0.785398
  - 0.0
  - -2.35619
  - 0.0
  - 1.5708
  - 0.785398

# OMPL planner ID
planner_id: RRTconnect

# OMPL optimal planner ID (used for smooth plan linear movement)
optimal_planner_id: RRTstar

# Offset from the face of the object to the center of the EEF
eef_offset: 0.095

# Movement scale constraints
max_acceleration_scale: 0.05
max_velocity_scale: 0.7

# Linear Mover type (cartesian, smoothplan)
linear_mover: smoothplan

# Cartesian movement
safe_linear: false
linear_eef_step_size: 1.5
linear_jump_threshold: 0.0000
linear_first_point_fraction: 0.0003
linear_n_waypoints: 10
